File: help/pt_pt/exercicios_da_ficha_11/Ficha11_014/Ficha11_014.js
// Programador           -

//Main Function
function main() {
    document.write("Altura  : " );// print text
    altura = parseInt(prompt("Altura  : " ,"0") );
    
    document.write(altura + "\n");// print input 
    document.write("Largura : " );// print text
    largura = parseInt(prompt("Largura : " ,"0") );
    
    document.write(largura + "\n");// print input 
    caixa(largura , altura ) ;
}

//::::::::::::::::::::::::::::::::::::::::::::::::::
//User Defined Functions Code
//::::::::::::::::::::::::::::::::::::::::::::::::::
function linha(n) {
    for( var i = 0; i < n; i = i + 1 ) {
        document.write("*" );
    }
    document.write("\n" );
}
//::::::::::::::::::::::::::::::::::::::::::::::::::
function interior(n) {
    document.write("*" );
    for( var i = 1; i < n - 1; i = i + 1 ) {
        document.write(" " );
    }
    document.write("*\n" );
}
//::::::::::::::::::::::::::::::::::::::::::::::::::
function caixa(larg, alt) {
    linha(larg ) ;
    for( var i = 1; i < alt - 1; i = i + 1 ) {
        interior(larg ) ;
    }
    linha(larg ) ;
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:13

File: help/pt_pt/exercicios_da_ficha_11/Ficha11_014/Ficha11_014.cs
// Programador           -

using System.IO;
using System;
public class Ficha11_014 {

    //Main Function
    static void Main() {
        Console.Write("Altura  : " );
        int altura = int.Parse(Console.ReadLine());
        Console.Write("Largura : " );
        int largura = int.Parse(Console.ReadLine());
        caixa(largura , altura ) ;
    }

    //::::::::::::::::::::::::::::::::::::::::::::::::::
    //User Defined Functions Code
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void linha( int n){
        for( int i = 0; i < n; i = i + 1 ) {
            Console.Write("*" );
        }
        Console.Write("\n" );
    }
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void interior( int n){
        Console.Write("*" );
        for( int i = 1; i < n - 1; i = i + 1 ) {
            Console.Write(" " );
        }
        Console.Write("*\n" );
    }
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void caixa( int larg , int alt){
        linha(larg ) ;
        for( int i = 1; i < alt - 1; i = i + 1 ) {
            interior(larg ) ;
        }
        linha(larg ) ;
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:13
